rpc/tls.go

```diff
@@ -99,10 +110,22 @@ func LoadInsecureTLSConfig() *TLSConfig {
 }
 
 // LoadTestTLSConfig loads the test TLSConfig included with the project. It requires
-// a path to the project root.
+// a path to the project root, loading the certs from assets bundled with the test.
 // TODO Maybe instead of returning err, take a testing.T?  And move to tls_test?
-func LoadTestTLSConfig(projectRoot string) (*TLSConfig, error) {
-	return LoadTLSConfig(path.Join(projectRoot, "resource", "test_certs"))
+func LoadTestTLSConfig(certDir string) (*TLSConfig, error) {
```

Comment:
This function should also move to the package where the testing certs live (cockroach/rpc/rpctest?)